### DISCUSSION TITLE
Java/SE/0_IDEA%E5%AE%89%E8%A3%85%E4%B8%8E%E8%AE%BE%E7%BD%AE

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#671] [Announcements]
# Java/SE/0_IDEA%E5%AE%89%E8%A3%85%E4%B8%8E%E8%AE%BE%E7%BD%AE

[TOC] IDEA介绍 IDEA 全称 IntelliJ IDEA，是Java编程语言开发的集成环境。IDEA在业界，长期被公认为最好的Java开发工具，尤其在智能代码助手、代码自动提示、重构、JavaEE支持、各类版本工具(git、svn等)、JUnit、CVS整合、代码分析、 创新的GUI设计等方面的功能可以说是具有划时代领先的。IDEA开发公司...

https://lsauce.github.io/Java/SE/0_IDEA%E5%AE%89%E8%A3%85%E4%B8%8E%E8%AE%BE%E7%BD%AE.html

**Comment by VagrantDcx** [comment]
666
